packages/cli/src/commands/deploy/index.ts

```diff
@@ -280,7 +292,7 @@ export default async (client: Client): Promise<number> => {
   let { org, project, status } = link;
 
   let newProjectName = null;
-  let rootDirectory = project ? project.rootDirectory : null;
+  let rootDirectory = project?.rootDirectory || undefined;
```

Comment:
Are you sure `null` and `undefined` mean the same thing here?

Reply:
I'm pretty sure, but to be careful, I switched this back. I updated the types here to allow `null`, which matches the types in `vercel/api`.

[`bc66fa4` (#9805)](https://github.com/vercel/vercel/pull/9805/commits/bc66fa46bcaf600c0c0e1067dc3c987cd67cb3c5) + [`40d8fd6` (#9805)](https://github.com/vercel/vercel/pull/9805/commits/40d8fd6750e630b008e76e346a7f718f4e0c4429)